### PR TITLE
Add support for post data in accessTokenCreate, handle missing refresh token in refreshAccessToken

### PR DESCRIPTION
**Pull request by zenovich**

### CODE REVIEW

File: app/api/auth/refresh_access_token.go [hunk 1]
```diff
@@ -73,11 +80,16 @@ func (srv *Service) refreshAccessToken(w http.ResponseWriter, r *http.Request) s
...
+	newToken string, expiresIn int32, apiError service.APIError) {
 	var refreshToken string
-	service.MustNotBeError(
-		srv.Store.RefreshTokens().Where("user_id = ?", user.GroupID).
-			PluckFirst("refresh_token", &refreshToken).Error())
+	err := srv.Store.RefreshTokens().Where("user_id = ?", user.GroupID).
+		PluckFirst("refresh_token", &refreshToken).Error()
+	if gorm.IsRecordNotFoundError(err) {
+		logging.Warnf("No refresh token found in the DB for user %d", user.GroupID)
+		return "", 0, service.ErrNotFound(errors.New("no refresh token found in the DB for the authenticated user"))
```
[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/auth/refresh_access_token.go:90]: He we encounter that the access token is okay (we run this method via the UserMiddleware), but no refresh token for the user. So I think the message is correct

[the remark this answers is not in the document]
Comment (smadbe) [reply app/api/auth/refresh_access_token.go:90]: Yes okay, but is the bug which was causing the 500 when the access token was wrong fixed? Returning "no refresh token found in the DB for the authenticated user" in this case would be wrong as well.

[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/auth/refresh_access_token.go:90]: The 500 error was not caused by a wrong access token, but by absence of a refresh token. So the error message is correct.

[the remark this answers is not in the document]
Comment (zenovich) [reply app/api/auth/refresh_access_token.go:90]: > Yes okay, but is the bug which was causing the 500 when the access token was wrong fixed?

Yes, it is fixed.

**Pull request merged**